ibm/mas_devops/roles/ocp_login/tasks/login-aws.yml

```diff
@@ -0,0 +1,10 @@
+---
+# 1. aws Login
+# -----------------------------------------------------------------------------
+- name: "login-aws : Oc Login the created cluster"
+  vars:
+    login_password: "{{ lookup('env', 'OPENSHIFT_PASSWORD') | default('', True) }}"
```

Comment:
To align with the existing variable naming convention can you please use `OCP_PASSWORD`, `OCP_USERNAME` (e.g. to match `OCP_VERSION`

Reply:
done